src/ar/mangaae/src/eu/kanade/tachiyomi/extension/ar/mangaae/MangaAe.kt

```diff
@@ -160,27 +164,30 @@ class MangaAe : ParsedHttpSource(), ConfigurableSource {
     // ============================== Settings ==============================
     companion object {
         private const val RESTART_TACHIYOMI = ".لتطبيق الإعدادات الجديدة Tachiyomi أعد تشغيل"
```

Comment:
consider rephrasing it to not include app name

Reply:
Ok